### DISCUSSION TITLE
[Feature] Release possibility without dependency on docker

### DISCUSSION THREAD

**Discussion by jeroenh** [immich-app/immich#3377] [Feature Request]
### The feature

It would be great to have the ability to run Immich without depending on docker.
So that users who are already self-hosting a webserver, database and other productivity tools can easily install Immich without duplicating a lot of these services.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by zackpollard** [comment]
You can already use immich with any other database with the use of environment variables. The internal webserver would be required regardless, and you can use your own reverse proxy instead of nginx if you want to. These are all configurable from the docker compose.

**Reply by jeroenh** [reply]
thanks for the quick response!

**Comment by bo0tzz** [comment]
Duplicate of #1657
